### PR TITLE
chore: add lanelet2 map config

### PR DESCRIPTION
**Pull request by meliketanrikulu**
## Description
This PR aims to provide the ability to change lanelet2 map configurations from within autoware_launch.
At the same time, the following PR has been created in order to make changes to the config file in autoware.universe. Before this PR is merged, the PR in autoware.universe must be merged for the changes here to take effect.
Related PR: https://github.com/autowarefoundation/autoware.universe/pull/2670

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kminoda** [comment]
LGTM in terms of parameter configuration, but since I am not familiar with lanelet2, let me redirect the review to the person in charge.
@mitsudome-r Would you briefly check this PR?

File: autoware_launch/config/map/lanelet2_map_loader.param.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**:
+  ros__parameters:
+    lanelet2_map_projector_type: UTM   # Options: MGRS, UTM
```
Comment (kminoda) [inline autoware_launch/config/map/lanelet2_map_loader.param.yaml:3]: Why is this projector type is changed from the default parameter in autoware.universe/map/map_loader/config ?

**Review by kminoda — COMMENTED** [review]
I left one small comment :pray:

**Review by mitsudome-r — APPROVED** [review]

**Comment by mitsudome-r** [comment]
LGTM, could you modify the pr to pass the CI?

**Pull request merged**